### ISSUE TITLE
how to create mask for text-inpainting textdiffuser

### ISSUE THREAD

**Issue by cuongngm** [microsoft/unilm#1344]
**Describe**
Model I am using (UniLM, MiniLM, LayoutLM ...): textdiffuser

dear author, 
as title, i want to ask, how to create mask (store in assets/examples/text-inpainting/_mask.jpg) for text-inpainting textdiffuser? Is model work with the text in another language (other than English) and how to finetune it?

**Comment by JingyeChen** [comment]
Thanks for your attention to TextDiffuser. Since the input image is resize to 512x512, you can use some auxiliary tools (e.g., powerpoint) to design the mask. 

Currently TextDiffuser only supports English and you need to collect dataset in other language at first for fine-tuning.
